test_app/tests/feature_flags/models/test_aap_flag_audit_logging.py

```
@@ -0,0 +1,126 @@
+
+# -----------------------------------------------------------------------------
+# Audit log test: verify create generates audit log
+# -----------------------------------------------------------------------------
+
+
+@pytest.mark.django_db
+@pytest.mark.skipif(not ACTIVITYSTREAM_INSTALLED, reason="activitystream not installed")
+def test_audit_log_aap_flag_create():
+    """Creating an AAPFlag emits an audit log record at INFO."""
+    with mock.patch(AUDIT_LOG_PATCH) as log_auth_event:
+        flag = AAPFlag.objects.create(
```

Ruff (0.15.2)

[error] Local variable `flag` is assigned to but never used

Remove assignment to unused variable `flag`

(F841)